doppler/resource_sync_types.go

```diff
@@ -41,17 +41,24 @@ func resourceSyncAWSSecretsManager() *schema.Resource {
 				Optional:    true,
 				ForceNew:    true,
 			},
+			"use_doppler_suffix": {
+				Description: "If enabled, Doppler will append /doppler to the provided path.",
```

Comment:
ultranit/optional: Reword from "If enabled" to "If not disabled", since it's enabled by default?

Reply:
(Reworded by the force push, which changes to boolean to an enum for better backwards compatibility)